nixos/modules/services/networking/dnscrypt-proxy2.nix

```diff
@@ -38,7 +38,7 @@ in
         Path to TOML config file. See: <link xlink:href="https://github.com/DNSCrypt/dnscrypt-proxy/blob/master/dnscrypt-proxy/example-dnscrypt-proxy.toml"/>
       '';
       example = literalExample "/etc/dnscrypt-proxy/dnscrypt-proxy.toml";
-      type = types.nullOr types.path;
+      type = types.path;
```

Comment:
Wouldn't it be null if one chose not to use this optional setting?

Reply:
It seems to work without and @Infinisil mentioned it's unnecessary in their review of your PR.